index.html

```diff
@@ -2612,6 +2624,9 @@ <h3>Disputes</h3>
 }
         </pre>
 
+<p> In the above credential the issuer is claiming that the address in the disputed
+credential is wrong, for example, the latter might wrongly be claiming to have the same address as that of the issuer.
+</p>
```

Comment:
Fix formatting: indent `<p>` and `</p>` properly, flush text left, word wrap like rest of text.